jobsdb/readonly_jobsdb.go

```diff
@@ -398,7 +411,12 @@ func (jd *ReadonlyHandleT) getProcessedJobsDSCount(ctx context.Context, ds dataS
 	jd.logger.Debug(sqlStatement)
 
 	row := txn.QueryRow(sqlStatement, time.Now())
-	defer txn.Commit()
+	defer func() {
+		err := txn.Commit()
```

Comment:
same here, `readonly_jobsdb` only performs select statements against the database.